onnxruntime/core/providers/cuda/reduction/reduction_ops.cc

```diff
@@ -65,7 +64,11 @@ namespace cuda {
       T,                                                                                   \
       kCudaExecutionProvider,                                                              \
       (*KernelDefBuilder::Create()).TypeConstraint("T", DataTypeImpl::GetTensorType<T>()), \
-      name<T>);                                                                            \
+      name<T>);
+
+// Register those with changes in OpSet12.
```

Comment:
nit: comment for 13

Reply:
I think the aim of this comment is to say there is a change from version 11 to 12